src/adafruit_blinka/board/jetson_pin.py

```diff
@@ -0,0 +1,48 @@
+"""Pin definitions for Jetson models."""
+
+from adafruit_blinka.microcontroller.tegra import pin
+
+D1 = pin.D1
```

Comment:
please follow the naming conventions you have for the boards. e.g.
https://www.jetsonhacks.com/nvidia-jetson-tx2-j21-header-pinout/
`GPIO_GCLK` `GPIO5` rather than coming up with non-standard names that don't match the documentation.

there should be a `board` file for each different Jetson model, where the name `GPIO5` will match up to the chip's underlying true name, e.g. GPIO398

![image](https://user-images.githubusercontent.com/1214161/54076234-4e143780-4277-11e9-8e9f-16d9ee2a8129.png)